eox_tagging/settings/common.py

```diff
@@ -28,7 +28,7 @@
 ]
 
 EOX_AUDIT_MODEL_APP = 'eox_audit_model.apps.EoxAuditModelConfig'
-
+# pylint: disable=R0801
```

Comment:
Can we be more descriptive with this disable? Instead of using the code, we can use the name of the error to know what we are avoiding, e.g., duplicate-code.
Can we avoid the duplicate-code error in this case, or is there a reason we disable it?

Reply:
Thanks, the following adjustments were made:
- The disable was removed. 
- The code duplication problem was solved